scripts/deploy_bug.py

```diff
@@ -66,57 +72,63 @@ def main(argv):
 
     # Convert to int and drop any that don't
     tags = [int(tag) for tag in tags if tag.isdigit()]
+    return tags
+
+
+def main(argv):
+    # Fetch tags as numbers (e.g. 309)
+    tags = get_tags()
 
     # The current tag is the max of that
     current_tag = max(tags)
 
     # The next tag is current_tag + 1
     next_tag = current_tag + 1
 
+    # Figure out the current version
+    sha = fetch_current_revision(HOST)
+
+    # Get the commits between the currently deployed version and what's in
+    # master tip and if there's nothing to deploy, then we're done!
+    resp = fetch_history_from_github('mozilla-services', 'socorro', sha)
+    if resp['status'] != 'ahead':
+        print('Nothing to deploy!')
+        return
```

Comment:
This fixes the case where there's nothing to deploy. Previously, it'd throw a traceback because it'd try to operate on an empty list.